src/cljs/rems/applications.cljs

```diff
@@ -54,20 +83,34 @@
 
 ;;;; UI
 
+;; XXX: the application lists share sorting and filtering state
```

Comment:
If we change to a model where all application data fetch state is in under one path in db, we could put the filter state there too.